sdk/identity/azure-identity/src/main/java/com/azure/identity/package-info.java

```diff
@@ -268,8 +268,8 @@
  * authenticate with <a href="https://learn.microsoft.com/entra/fundamentals/">Microsoft Entra ID</a>.</p>
  *
  * <p>The Azure Identity library supports user credentials based authentication via
- * {@link com.azure.identity.InteractiveBrowserCredential}, {@link com.azure.identity.DeviceCodeCredential} and
- * {@link com.azure.identity.UsernamePasswordCredential}. For more information refer to the
+ * {@link com.azure.identity.InteractiveBrowserCredential} and {@link com.azure.identity.DeviceCodeCredential}.
```

Comment:
We only list these 2 creds, but the link below takes you to a section with 4 creds. It's probably best to remove this specific line and let the link show the reader which creds support this scenario.